### PR TITLE
Update main.go

### PR DESCRIPTION
**Pull request by ferreiratiago**

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -5,3 +5,6 @@ import "fmt"
 func main() {
 	fmt.Println("Hello, world!  x2")
 }
+
```
Comment (ferreiratiago) [inline main.go:8]: Why?

**Review by ferreiratiago — COMMENTED** [review]
Comment body

File: main.go (hunk 1 above)
Comment (padawancat) [inline main.go:8]: Why?

**Review by padawancat — CHANGES_REQUESTED** [review]
Body of why

**Pull request closed without merging**